ceremonies/src/lib.rs

```diff
@@ -529,7 +783,7 @@ impl<T: Config> Module<T> {
             let meetup_count = Self::meetup_count((cid, cindex));
             let reward = Self::nominal_income(cid);
 
-            for m in 1..=meetup_count {
+            for m in 0..meetup_count {
```

Comment:
please respect the rule that indexing begins with 1
This is motivated by the fact that chain state doesn't store zeros, so you cannot know if meetup count is indeed zero or just hasn't been set.
Leave indexing rules as they are

Reply:
done